charts/Pie.args.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Viz from "../primitives/Viz";
+
+import {Pie as D3plusPie} from "d3plus-react";
+export const Pie = ({config}) => <D3plusPie config={config} />;
+
+export const argTypes = {
+
+    /**
+     * Filters out unused argTypes from the Viz primitive and
+     * overrides any defaults that have been changed in Treemap
+     */
+    ...Object.keys(Viz.argTypes)
+      .filter(k => !k.match(/^(discrete|shape|zoom.*)$/))
+      .reduce((obj, k) => (obj[k] = Viz.argTypes[k], obj), {}),
+  
+    /**
+     * Pie-specific methods
+     */
+     innerRadius: {
+      type: {
+        summary: "function | number"
+      },
+      table: {
+        defaultValue: {
+          summary: "function | number",
+          detail: "0"
+        }
+      },
+      description: `The function or value to be used as inner radius in the Pie.`
+    },
+    padAngle: {
+      type: {
+        summary: "number"
+      },
+      table: {
+        defaultValue: {
+          summary: "number",
+          detail: "0"
+        }
+      },
+      control: {
+        type: "number"
+      },
+      description: `The arc padding to the specified radian value between \`0\` and \`1\`.
+      
+By default the value is \`0\`, without setting any padding between each arc.`
+    },
```

Comment:
* remove defaultValue "detail" and set summary to `"undefined"`
* description:

The padding between each arc, set as a radian value between \\\`0\\\` and \\\`1\\\`.

If set, this will override any previously set padPixel value.